public/style/components/_status.scss

```diff
@@ -0,0 +1,4 @@
+.status-primary {
```

Comment:
Continuing the thought of merging this into a single file with the other `forecast` and `data` partial.